packages/angular/cli/commands/update-impl.ts

```diff
@@ -177,12 +175,45 @@ export class UpdateCommand extends SchematicCommand<UpdateCommandSchema> {
       } else if (typeof migrations !== 'string') {
         this.logger.error('Package contains a malformed migrations field.');
 
+        return 1;
+      } else if (path.posix.isAbsolute(migrations) || path.win32.isAbsolute(migrations)) {
```

Comment:
Why not `path.isAbsolute`? in windows `/foo/abc` is still considered as absolute path.

Reply:
I wanted to ensure that it errors on all cases of an absolute path on all systems.  Windows has `C:`,`\\`, etc.

Reply:
![image](https://user-images.githubusercontent.com/17563226/58885512-a4f8a400-86e2-11e9-88c2-6fea137ec9d8.png)